lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -19,17 +20,42 @@ function canFindDenominatorInNumerator(node) {
   if (Node.Type.isParenthesis(denominator)) {
     denominator = denominator.content;
   }
+  if (!(numerator.op === '+' || numerator.op === '-' ||
+        denominator.op === '+' || numerator.op === '-')) {
+    return false;
+  }
+  if (denominator.op !== '+') {
+    return false;
+  }
+
+  let numeratorFirstTerm;
+  if (numerator.op === '+') {
+    numeratorFirstTerm = new Node.PolynomialTerm(numerator.args[0]);
+  }
+  else if (numerator.op === '*') {
+    numeratorFirstTerm = new Node.PolynomialTerm(numerator);
+  }
+
+  let denominatorFirstTerm;
+  if (denominator.op === '+') {
+    denominatorFirstTerm = new Node.PolynomialTerm(denominator.args[0]);
+  }
+  else if (denominator.op === '*') {
+    denominatorFirstTerm = new Node.PolynomialTerm(denominator);
+  }
 
-  if (numerator.op !== '+') {
+  if (!(numeratorFirstTerm)) {
     return false;
   }
-  if (!('args' in denominator)) {
+  if (!(denominatorFirstTerm)) {
+    return false;
+  }
+
+  if (!(numeratorFirstTerm.getSymbolName() === 'x' && denominatorFirstTerm.getSymbolName() === 'x')) {
```

Comment:
using `getSymbolName` makes sure the symbol in numerator and denominator are the same - i.e. both `x` or both `y` or whatever - you could take any example with `x`s and change them to `y`s and it should also work (actually can you add a test for that? ^_^)

`numeratorFirstTerm.getSymbolName() === denominatorFirstTerm.getSymbolName()` is what I was thinking